### DISCUSSION TITLE
How to add new class name or change class name in 'AdminTabs' shapes?

### DISCUSSION THREAD

**Discussion by ChestnutYYY** [OrchardCMS/OrchardCore#11537] [Q&A] [answered]
Recently, I‘ve overridden 'AuditTrailSettings.Edit.cshtml' page, but i can not find  where are tabs. According to class name, 
 I find the tabs from ‘GroupShapes --AdminTabs method’ maybe, 
Now, I write js ‘$(".nav .nav-tabs").addClass("mytestclass")’  in ‘AuditTrailSettings.Edit.cshtml’ page
  I would like to know, way such as overring  'GroupShaps.cs' or something else?

Thanks~
![image](https://user-images.githubusercontent.com/13895891/163309012-8fc518b2-4f8d-41d5-acb1-7731da52ddb4.png)

**Comment by ns8482e** [comment, marked as answer]
@rllxxx `AdminTabs` is a shape that builds tabs. You  can override it by creating `AdminTabs.cshtml`

**Reply by ChestnutYYY** [reply]
okey，I tried it and worked. thanks you so much~
